docs/src/guides/posting-transactions/turbo.md

```diff
@@ -101,14 +106,21 @@ console.log("Result", uploadResult);
 ```js
 const filePath = new URL('path/to/file', import.meta.url).pathname;
 const fileSize = fs.statSync(filePath).size;
+const dataItemOpts = {
+    //target: 'string',
+    // anchor: 'string',
+    tags: [{name: 'test', value: 'test'}] // add tags
+  }
 const uploadResult = await turbo.uploadFile({
 	fileStreamFactory: () => fs.createReadStream(filePath),
 	fileSizeFactory: () => fileSize,
-	signal: AbortSignal.timeout(10_000), // cancel the upload after 10 seconds
+	signal: AbortSignal.timeout(10_000), // Optional: cancel the upload after 10 seconds
+  dataItemOpts // Optional
 });
 console.log(JSON.stringify(uploadResult, null, 2));
```

Comment:
Good practice to add a code snippet in the `console` or `bash` language with the expected output here

Reply:
Added